package.json

```diff
@@ -58,6 +59,7 @@
     "build-test": "rm -rf dist-tests && cup build-tests",
     "just-test": "node_modules/.bin/unitest --browser=dist-tests/browser.js --node=dist-tests/node.js",
     "test": "npm run build-test && npm run just-test",
+    "cover": "npm run build-test && nyc npm run just-test",
```

Comment:
Seems like this is the exact same target as `yarn test`? Why do we need it?

Reply:
adds `nyc` for coverage